flask_common/documents.py

```diff
@@ -75,6 +75,12 @@ def save(self, *args, **kwargs):
             self.date_updated = now
         return super(DocumentBase, self).save(*args, **kwargs)
 
+    def modify(self, *args, **kwargs):
```

Comment:
If `upsert` is True, this should also set `date_created`. I'm fine with also throwing `NotImplementedError` for upserts though.